services/explorer/src/pages/BlockData/DAGDataContainer.tsx

```diff
@@ -98,21 +98,23 @@ const BlockDataContainer = (): JSX.Element => {
               <HashLink
                 linkType={HashLinkType.ADDRESSES}
                 width='auto'
-                hash={blockData?.sender}
+                hash={zeroX(blockData?.author?.address)}
```

Comment:
Are you sure we need to add '0x' in front of the address?

Reply:
I was adding it just for displaying, but you're right, I'm moving it into the displaying component